modules/vstudio/vs2019.lua

```diff
@@ -28,8 +28,8 @@
 		valid_kinds     = { "ConsoleApp", "WindowedApp", "StaticLib", "SharedLib", "Makefile", "None", "Utility", "SharedItems" },
 		valid_languages = { "C", "C++", "C#", "F#" },
 		valid_tools     = {
-			cc     = { "msc"   },
-			dotnet = { "msnet" },
+			cc     = { "msc", "clang" },
+			dotnet = { "msnet"        },
```

Comment:
I don't believe we do this kind of thing anywhere else, can you revert this?

Reply:
Yup. I'll have those changes out for review tomorrow evening (eastern).

Reply:
Sorry, I think there might have been a misunderstanding here. I was just referring to the whitespace in the `dotnet` line. It looks like you've reverted the `cc` change too?

Reply:
Yeah, I think I misunderstood.  It looked like the code was trying to keep the brackets in line, so that's what I was doing there.  I'll just put back the clang and do nothing with bracket alignment, if that works.

Reply:
I might be wrong on the spacing, but I don't recall seeing that anywhere else. Thanks for that, and sorry about the confusion here.